config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -348,7 +348,7 @@ periodics:
       - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
       - --node-tests=true
       - --provider=gce
-      - --test_args=--nodes=8 --skip="\[Flaky\]|\[Serial\]"
+      - --test_args=--nodes=8 --skip="\[Flaky\]|\[Serial\]|\[NodeAlphaFeature:.+\]"
```

Comment:
Instead of doing this, can you change the job to focus on `[NodeConformance]` instead?
Starting from 1.11, we split the tests into "NodeConformance" and "NodeFeature", etc. I can add a NodeFeature job later.